app/dashboard/utils.py

```diff
@@ -55,15 +55,29 @@ class NoBountiesException(Exception):
     pass
 
 
-def humanize(text):
-    """Clean a notification string of underscores and capitalize it.
+def humanize_event_name(name):
```

Comment:
Can you write a test for this method in `dashboard/tests/test_dashboard_utils.py` ?